### PR TITLE
ship content strings locales

### PR DESCRIPTION
**Pull request by deepak1556**

### CODE REVIEW

File: script/create-dist [hunk 1]
```diff
@@ -331,6 +333,16 @@ def copy_generated_sources(target_arch, component, output_dir):
                  relative_to=os.path.join(output_dir, config, 'gen'),
                  destination=os.path.join(target_dir, 'gen'))
 
+def copy_locales(target_arch, component, output_dir):
+  config = get_configuration(target_arch)
+  target_dir = os.path.join(MAIN_DIR, component, 'locales')
+  mkdir_p(target_dir)
+  src_dir = os.path.join(output_dir, config, 'gen', 'content', 'app', 'strings')
+  for dirpath, dirname, filenames in os.walk(src_dir):
```
Comment (zcbenz) [inline script/create-dist]: We should just use `glob`.

File: script/create-dist [hunk 2]
```diff
@@ -331,6 +333,16 @@ def copy_generated_sources(target_arch, component, output_dir):
...
+    for filename in filenames:
+        new_name = re.sub('content_strings_', '', filename)
```
Comment (zcbenz) [inline script/create-dist]: 2 spaces indention.

**Comment by deepak1556** [comment]
@zcbenz Fixed, thanks!

**Comment by zcbenz** [comment]
:+1:

**Pull request merged**